packages/front-end/pages/saved-groups.tsx

```diff
@@ -114,12 +131,30 @@ export default function SavedGroupsPage() {
         </div>
       )}
       {savedGroups.length === 0 && (
-        <div className="alert alert-info">
-          You don&apos;t have any saved groups defined yet.{" "}
-          {permissions.manageSavedGroups && (
-            <span>Click the button above to create your first one.</span>
-          )}
-        </div>
+        <>
+          <p className="mb-3">
+            Saved Groups are defined set of attribute values which can be used
+            with feature rules for targeting features at particular users. For
+            example, you might create a list of internal users.
+          </p>
+          <div className="alert alert-info mb-2">
+            You don&apos;t have any saved groups defined yet.{" "}
+            {permissions.manageSavedGroups && (
+              <span>Click the button above to create your first one.</span>
+            )}
+          </div>
+        </>
+      )}
+      {auditModal && (
+        <Modal
+          open={true}
+          header="Audit Log"
+          close={() => setAuditModal(false)}
+          size="max"
+          closeCta="Close"
+        >
+          <HistoryTable type="savedGroup" id={"all"} />
```

Comment:
Instead of passing in `"all"`, we can make `id` an optional prop and change which API endpoint we hit depending on whether or not it's provided.

Reply:
done